media_driver/linux/common/ddi/media_libva.cpp

```diff
@@ -3915,6 +3915,7 @@ static VAStatus DdiMedia_SyncSurface (
     return DdiMedia_StatusCheck(mediaCtx, surface, render_target);
 }
 
+#if VA_CHECK_VERSION(1, 8, 0)
```

Comment:
sorry, but looking into https://github.com/intel/libva/blob/master/va/va_version.h.in#L82 it probably need to be VA_CHECK_VERSION(1, 9, 0). Can you, please, double check?